app/graphql/types/task_type.rb

```diff
@@ -0,0 +1,15 @@
+module Types
+  class TaskType < Types::BaseObject
+    field :id, ID, null: false
+    field :name, String, null: false
+    field :done, Boolean, null: false
+    field :created_at, String, null: false
+    field :project, Types::ProjectType, null: false
+    field :comments, [Types::CommentType], null: false
+    field :comments_count, Integer, null: true
```

Comment:
it could be `null: false` as `object.comments.size` will return 0 if there is no comments